ObjectsComparer/ObjectsComparer/Enums/ComparisonStatus.cs

```diff
@@ -12,8 +12,8 @@ namespace ObjectsComparer.Enums
   public enum ComparisonStatus
```

Comment:
I was thinking about removing this enum completely. It simplifies things a lot.

Reply:
Ok. I have removed it.